app/controllers/resource_controller.rb

```diff
@@ -115,7 +115,7 @@ def redirect_after_save
   end
 
   def search_resources
-    resource_class
+    resource_class.all
```

Comment:
why was this necessary ?

Reply:
Well this is optional for paginated resources, but for non-paginated resource, we need to list all the resources.
While during pagination this will get overridden by pagy.

Reply:
prefer to put that into the resoruce_controller so it's fixed in general